### ISSUE TITLE
SRT results

### ISSUE THREAD

**Issue by flynnamy** [D-X-Y/landmark-detection#82]
Hi，I pull newest code,the results is not accurate. You said it works well. Please see my result is same with yours! @D-X-Y 
![image](https://user-images.githubusercontent.com/20398890/105795428-c9944480-5fc7-11eb-9373-9dfcab57abb4.png)

**Comment by D-X-Y** [comment]
Yes, the prediction is reasonable while not that accurate. First, this is predicted by the regression model (much simpler than a heatmap-based model, please see metrics in our paper). Second, this pic is for demo, the image style and crop size are very different from the training data. If you expect an accurate results, you can try on 300W.
